samples/parallel-processing-california-housing-data/README.md

```diff
@@ -0,0 +1,140 @@
+# Overview
+
+**What does this sample demonstrate:**
+- A simple example of how to use `ParallelRunScript` to process data in parallel.
```

Comment:
The whole sample is to demonstrate how efficient the parallel processing step is in these kind of scenarios. Would be great if you could explain here what is done in parallel and that we experienced up to x % time reduction.

Reply:
updated in the new commit https://github.com/liupeirong/MLOpsManufacturing/pull/69/commits/5a6dbbf4d8bc97a67a37db14f1a99dc50637c5e2.